test/parallel/test-vm-getter-throw.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
```

Comment:
Could you include a brief description of what this test does (as per the [guide](https://github.com/nodejs/node/blob/master/doc/guides/writing-tests.md#lines-4-5))?